applications/GeoMechanicsApplication/custom_elements/three_dimensional_stress_state.h

```diff
@@ -0,0 +1,34 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Marjan Fathian
+//                   Richard Faasse
+//
+
+#pragma once
+
+#include "stress_state_policy.h"
+
+namespace Kratos
+{
+
+class ThreeDimensionalStressState : public StressStatePolicy
+{
+public:
+    [[nodiscard]] Matrix CalculateBMatrix(const Matrix&         GradNpT,
+                                          const Vector&         Np,
+                                          const Geometry<Node>& rGeometry) const override;
```

Comment:
Does this follow the naming conventions?

Reply:
Good catch. The parameter names have been updated.